packages/salesforcedx-vscode-apex-replay-debugger/package.json

```diff
@@ -144,18 +140,6 @@
           "when": "view == sfdx.force.test.view && viewItem =~ /(apexTest)(_.*|\\b)/",
           "group": "inline"
         }
-      ],
-      "editor/context": [
-        {
-          "command": "sfdx.launch.replay.debugger.logfile",
-          "when": "sfdx:project_opened && editorLangId == 'apexlog'"
-        }
-      ],
-      "explorer/context": [
-        {
-          "command": "sfdx.launch.replay.debugger.logfile",
-          "when": "sfdx:project_opened && resourceIsFile && resourceLangId == 'apexlog'"
-        }
       ]
```

Comment:
I removed the `sfdx.launch.replay.debugger.logfile` command and consolidated it into the "Launch Apex Replay Debugger with Current File" command (`sfdx.force.launch.apex.replay.debugger.with.current.file`, in packages/salesforcedx-vscode-core)